### PR TITLE
[OneExplorer] Reveal focused cfg file in OneExplorer

### PR DESCRIPTION
**Pull request by dayo09**
This commit reveals focused cfg file on CfgEditor in OneExplorer.

ONE-vscode-DCO-1.0-Signed-off-by: Dayoung Lee <dayoung.lee@samsung.com>

----

From #1246

### GIF

0913-reveal

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -147,7 +147,12 @@
         "category": "ONE"
       },
       {
-        "command": "one.explorer.reveal",
+        "command": "one.explorer.revealInOneExplorer",
+        "title": "Reveal in ONE Explorer View",
+        "category": "ONE"
+      },
+      {
+        "command": "one.explorer.revealInDefaultExplorer",
```
Comment (dayo09) [inline package.json:155]: NOTE: I changed the name 'one.explorer.reveal' command into 'revealInDefaultExplorer' to distinguish it from 'revealInOneExplorer' command.

File: src/CfgEditor/CfgEditorPanel.ts [hunk 2]
```diff
@@ -175,6 +175,9 @@ export class CfgEditorPanel implements vscode.CustomTextEditorProvider {
     });
 
     webviewPanel.onDidChangeViewState(e => {
+      if (webviewPanel.visible) {
```
Comment (dayo09) [inline src/CfgEditor/CfgEditorPanel.ts:178]: When focused document is changed in CfgEditorPanel, it's shown in OneExplorer's tree view.

File: src/OneExplorer/OneExplorer.ts [hunk 3]
```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+import {getLatestInsidersMetadata} from '@vscode/test-electron/out/util';
```
Comment (dayo09) [inline src/OneExplorer/OneExplorer.ts:17]:
```suggestion
```

File: src/OneExplorer/OneExplorer.ts [hunk 4]
```diff
@@ -468,7 +469,9 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<Node> {
 
   private fileWatcher = vscode.workspace.createFileSystemWatcher(`**/*`);
 
-  private tree: DirectoryNode|undefined;
+  private _tree: DirectoryNode|undefined;
```
Comment (dayo09) [inline src/OneExplorer/OneExplorer.ts:472]: NOTE: I changed tree -> _tree for code cleaning.

**Comment by dayo09** [comment]
@Samsung/one-vscode PTAL :-D

**Review by yunjayh — APPROVED** [review]
LGTM

**Comment by dayo09** [comment]
@Samsung/one-vscode PTAL :-D

**Review by jyoungyun — APPROVED** [review]
LGTM

**Pull request merged**